Add Logout button to the menu bar

maraPislar opened the pull request:
I added to the frontend the **following changes**:

- added logout button to the menu bar
- added some text to the login page

**Preview**:

NOTE: normal button

![1](https://user-images.githubusercontent.com/37437735/93599770-49315780-f9c7-11ea-92f9-22cfcace3959.png)

NOTE: When hovering over the button

![2](https://user-images.githubusercontent.com/37437735/93599759-48002a80-f9c7-11ea-9805-ed0bd87a5dd6.png)

![3](https://user-images.githubusercontent.com/37437735/93599751-46cefd80-f9c7-11ea-8ed2-32b1bec9a95b.png)

maraPislar committed:
Update README.MD

maraPislar committed:
Update README.MD

maraPislar committed:
Update README.MD

maraPislar committed:
Update README.MD

Commit:
Add Logout button and develope login page.

Krechals commented on README.MD:
```diff
@@ -1 +1,47 @@
 # GPostCard
+
+### Project Description
+
+GPostcard is a Web application that can generate postcards based on the text that is on them. After a postcard is generated, it can be sent to any Gmail user. By ticking the location, the user can get on the postcard a representative photo for their current location. Moreover, the user can record a message which will be automatically filled into the text area allowing accessability. 
```
There is this PR https://github.com/googleinterns/step230-2020/pull/67 that deals with the README file. Let's focus there on this file.

maraPislar replied:
Oops, of course. Sorry!

Commit:
Fix review issue.

Commit:
Delete README for focusing on another feature.

AndreMB0 reviewed (changes requested):
Nice, but since the login is via SSO, it would be useful to have the userEmail written as well (above/below/next to the logout button in the menu bar)

Commit:
Add email feature to the login class.

Commit:
Also retunr the email to be fetched.

Commit:
Add email next to the logout link

Commit:
Add style to the email text.

maraPislar commented:
I added the email next to the Logout link. Also added a left boarder for separating the links and for design.
**Preview**:

![Screenshot 2020-09-18 at 8 14 20 PM](https://user-images.githubusercontent.com/37437735/93626099-98897f00-f9eb-11ea-9695-b7c9514f57d2.png)

Commit:
Fix small issue.

Commit:
Add design detail: border between links for design purposes.

Commit:
Add more style to the links.
